src/index.pug

```diff
@@ -44,6 +44,68 @@ block content
                 Instead, Stryker empowers you to use what you like and focusses on being the best Mutation testing framework it can be.
     .row
         .col-md-12.text-center
+            :markdown-it
+
+                ## An example
+
+                Say you're building an online casino, users can only enter the casino when they're over 18. So you write the following piece of code to check if someone is allowed to use the website:
+
+                ```js
+                function isUserOldEnough(user) {
+                  return user.age >= 18;
+                }
+                ```
+
+                Stryker will find the return statement and decide to change it in a number of ways:
+                ```js
+                /* 1 */ return user.age > 18;
+                /* 2 */ return user.age < 18;
+                /* 3 */ return false;
+                /* 4 */ return true;
+                ```
+
+                We call those modifications mutants. After the mutants have been found, they are applied one by one and your tests will be executed against them. If at least one of your tests fail, we say the mutant is *killed*. That's what we want! If no tests fail, it *survived*. The better your tests, the less mutants survive.
+    .row
+        .col-md-12.text-center
+            :markdown-it
+
+                ## Output
+
+                Stryker will output the results in various different formats. One of the easiest output to read is via the clear text reporter:
+                ```shell
+                [DEBUG] ClearTextReporter - Mutant killed!
```

Comment:
This kinda overwhelms the reader a little bit. Could you maybe shorten it to one example? Maybe without the `[DEBUG] ClearTextReporter` part.